lib/tasks/cache.rake

```diff
@@ -0,0 +1,7 @@
+namespace :cache do
+  desc "Clear cache for a given base path"
+  task :clear, [:base_path] do |_, args|
```

Comment:
I wonder if it might be nicer to have a separate `cache:clear_varnish` task which the `cache:clear` task depends on. Then we can add Fastly as a new task later.

Reply:
I was thinking we should do that when fastly is added, so we can have `clear` which invokes `clear_varnish` and `clear_fastly`.

Reply:
Yep, that sounds good too 👍